presidio-anonymizer/setup.cfg

```diff
@@ -1,2 +1,5 @@
 [flake8]
-max-line-length = 88
\ No newline at end of file
+max-line-length = 88
+
+;remove requirement for init documentation and module
+extend-ignore = D100,D104,D107
```

Comment:
@shiranr why adding those?

Reply:
We need to discuss this in our daily/kickoff today. I think this should be a discussion.
IMHO there is such thing as too much comments, I don't believe in adding a comment so the linter will be happy.
A comment should be justified.

Reply:
Sure, let's investigate and talk, a random list here is not justified either...